packages/application-generic/src/services/readiness/readiness.service.ts

```diff
@@ -58,13 +50,15 @@ export class ReadinessService {
 
   private async checkServicesHealth() {
     try {
-      const healths = await Promise.all([
-        this.standardQueueServiceHealthIndicator.isHealthy(),
-        this.workflowQueueServiceHealthIndicator.isHealthy(),
-        this.subscriberProcessQueueHealthIndicator.isHealthy(),
-      ]);
+      const healths = await Promise.all(
+        this.indicators.map((indicator) => indicator.isHealthy())
+      );
+
+      const statuses = healths.map(
+        (health: HealthIndicatorResult) => Object.values(health)[0].status
+      );
 
-      return healths.every((health) => !!health === true);
+      return statuses.every((status: HealthIndicatorStatus) => status === 'up');
```

Comment:
Created this explicit check of the status.
The old way works now because we throw exceptions if the status `false`, however, i would like to be explicit if possible.